test/py/Makefile.am

```diff
@@ -117,6 +121,12 @@ gen-py-enum/%/__init__.py: ../%.thrift $(THRIFT)
 	&& $(THRIFT) --gen py:enum -out gen-py-enum ../v0.16/$(notdir $<) \
 	|| $(THRIFT) --gen py:enum -out gen-py-enum $<
 
+gen-py-type_hints/%/__init__.py: ../%.thrift $(THRIFT)
+	test -d gen-py-type_hints || $(MKDIR_P) gen-py-type_hints
+	test ../v0.16/$(notdir $<) \
+	&& $(THRIFT) --gen py:type_hints -out gen-py-type_hints ../v0.16/$(notdir $<) \
```

Comment:
UUIDs are handled (at least from what i see above) but you still want to test against the old uuid-free version?

Reply:
@arkuhn  Would like to merge this but ...